views/register.hbs

```diff
@@ -0,0 +1,52 @@
+<div class="login">
+    <section>
+        <div class="container">
+            <div class="login__form">
+                <div class="flex_box">
+                    <div class="form_text">
+                        <h2 tabindex="0">Login</h2>
+                        <p tabindex="0">Get access to your Orders , Wishlists and
+                            Recommendations </p>
+                    </div>
+                    <div class="form_set">
+                        <form autocomplete="off">
+                            <div class="fieldset">
+                                <input type="text" class="form-control"
+                                    id="First_name" required>
+                                <span class="border_bottom"></span>
+                                <label for="First_name">First Name</label>
+                            </div>
+                            <div class="fieldset">
+                                <input type="text" class="form-control"
+                                    id="Last_name" required>
+                                <span class="border_bottom"></span>
```

Comment:
Why use a separate TAG for creating border??

Reply:
Done